.ado/pipelines/templates/jobs-configuration.yaml

```diff
@@ -61,9 +64,7 @@ jobs:
                 --values src/config/ingress-nginx/values.helm.yaml `
                 --set controller.service.loadBalancerIP="$aksIngressIp" `
                 --set controller.service.annotations."service\.beta\.kubernetes\.io/azure-load-balancer-resource-group"="$aksClusterResourceGroup" `
-                --version "$(ingressNginxVersion)" `
-                --set controller.image.repository="$($globalInfraDeployOutput.acr_login_server.value)/$($container.image)" `
-                --set controller.image.tag="$($container.tag)"
+                --version "$(ingressNginxVersion)" $privateRegistryParam
```

Comment:
@sebader thanks! I prefer that approach - does it work now? Or have you just reverted it for testing purposes?